spec/ParseAPI.spec.js

```diff
@@ -429,6 +429,47 @@ describe('miscellaneous', function() {
     });
   });
 
+  it('test beforeSave get dirtyKeys on update', function(done) {
+    var triggerTime = 0;
+    // Register a mock beforeSave hook
+    Parse.Cloud.beforeSave('GameScore', function(req, res) {
```

Comment:
How is this going to interact with cloud code in other tests? If every cloud code test is modifying some global that seems like it could get messy fast.